### PR TITLE
Adding a lang attribute to the html element

### LINKED ISSUE #898: Increase Lighthouse Performance of Susper
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

image

**Current behavior:**
- Progressive web app
  - [x] The App does not provide fallback content when Javascript is not available.
  - [x] User is not prompted to install the Web App ( prompt for adding to home screen)
  - [x] Adding manifest.
  - [x] Address bar does not match brand colors
- Performance
  - [ ]  Does not Reduce Render-blocking script.
  - [ ]  Does not Reduce Render-blocking stylesheets.
- Accessibility
  - [ ]  Image elements do not have [alt] attributes.
  - [ ]  Form elements do not have labels.
- Best Practices
  - [ ]  Does not Opens External Anchors Using rel="noopener" .

**Expected behavior:**
All the Issue mentioned above should be resolved.

I am working on these Issue. I have made a checklist, will resolve each issue one by one 👍 
Please give suggestions @mariobehling  @harshit98 @nikhilrayaprolu

### PR DESCRIPTION
**Pull request by subhahu123**
Fixes #898 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- screenshot-dequeuniversity com-2019 01 27-16-37-05

- https://pr-1442-fossasia-susper.surge.sh

<a href="https://dequeuniversity.com/rules/axe/2.2/html-lang?application=lighthouse" > Reference </a>

### CODE REVIEW

**Review by praveenojha33 — APPROVED** [review]

**Comment by praveenojha33** [comment]
@AakashMallik @shreyanshdwivedi Please review.

File: src/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (Pipe-Runner) [inline src/index.html:2]: Could you justify the need for this change?

**Review by Pipe-Runner — CHANGES_REQUESTED** [review]

**Comment by subhahu123** [comment]
@AakashMallik please checkout the above given reference

File: src/index.html (hunk 1 above)
Comment (subhahu123) [reply src/index.html:2]: https://dequeuniversity.com/rules/axe/2.2/html-lang?application=lighthouse

**Review by shreyanshdwivedi — APPROVED** [review]
LGTM :+1:

File: src/index.html (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/index.html:2]: I think this reference justifies your change :) @subhahu123

Comment (Pipe-Runner) [reply src/index.html:2]: @subhahu123 Cool :+1: Good catch.

**Review by Pipe-Runner — APPROVED** [review]

**Review by sakshee-19 — APPROVED** [review]

**Review by bhaveshAn — APPROVED** [review]

**Pull request merged**